custom_components/bmw_connected_drive/device_tracker.py

```diff
@@ -33,9 +33,11 @@ def __init__(self, account, vehicle):
         self._vehicle = vehicle
         self._unique_id = vehicle.vin
         self._location = (
-            vehicle.state.gps_position if vehicle.state.gps_position else (None, None)
+            vehicle.state.vehicle_status.gps_position if vehicle.state.vehicle_status.gps_position else (None, None)
         )
         self._name = vehicle.name
+        self._heading = vehicle.state.vehicle_status.gps_heading
+        self._vin = vehicle.vin
```

Comment:
This is not used anymore.

Reply:
Yep I know :D thought I'd got rid of this...